.github/workflows/JuliaNightly.yml

```
@@ -13,16 +13,14 @@ jobs:
         with:
           version: nightly
           arch: x64
-      - uses: actions/cache@v1
+      - uses: actions/cache@v2
         env:
-          cache-name: cache-artifacts
+          cache-name: julianightly-cache-artifacts
```

I believe this is what you're looking for?

```suggestion
          cache-name: julia-nightly-cache-artifacts
```